README.md

```diff
@@ -3,8 +3,12 @@
 [Amber](https://github.com/Amber-Crystal/amber) is a web framework written in
 the [Crystal](https://github.com/manastech/crystal) language.
 
-This project is to provide an ORM Model in Crystal.
+This project is to provide an ORM in Crystal.
 
+  <p align="center">
```

Comment:
Can we use markdown instead? 😅 

Something like:

`[![Build Status](https://img.shields.io/travis/amberframework/granite-orm.svg?maxAge=360)](https://travis-ci.org/amberframework/granite-orm)`

WDYT?

Reply:
I had it as markdown first, but other projects had the badge centered on the readme. I'm up for either.